tests/test_filters_minicircle.py

```diff
@@ -47,16 +47,38 @@ def test_remove_x_y_tilt_unmasked(
     return fig
 
 
+@pytest.mark.mpl_image_compare(baseline_dir="resources/img/")
+def test_remove_quadratic_unmasked(
+    minicircle_initial_quadratic_removal: Filters, plotting_config: dict, plot_dict: dict, tmp_path
+) -> None:
+    """Test removal of quadratic with mask."""
+    assert isinstance(minicircle_initial_quadratic_removal.images["initial_quadratic_removal"], np.ndarray)
+    assert minicircle_initial_quadratic_removal.images["initial_quadratic_removal"].shape == (1024, 1024)
+    assert minicircle_initial_quadratic_removal.images["initial_quadratic_removal"].sum() == pytest.approx(
+        -1672538.8578738687
+    )
+    plotting_config = {**plotting_config, **plot_dict["initial_quadratic_removal"]}
+    fig, _ = Images(
+        data=minicircle_initial_quadratic_removal.images["initial_quadratic_removal"],
+        output_dir=tmp_path,
+        pixel_to_nm_scaling=minicircle_initial_quadratic_removal.pixel_to_nm_scaling,
+        **plotting_config,
+    ).plot_and_save()
+    return fig
+
+
 def test_get_threshold_otsu(minicircle_threshold_otsu: np.array) -> None:
     """Test calculation of threshold."""
     assert isinstance(minicircle_threshold_otsu.thresholds, dict)
-    assert minicircle_threshold_otsu.thresholds["upper"] == 28.58495414588038
+    assert minicircle_threshold_otsu.thresholds["upper"] == pytest.approx(-0.7471511148088736)
```

Comment:
I wouldn't have expected it as this is `test_get_threshold_otsu()` and so there should only be one threshold, which by default is stored with the `upper` key and so `lower` shouldn't have an influence here I don't think.

Reply:
I think it's because `minicircle_threshold_otsu` uses the `minicircle_initial_tilt_removal` fixture which has had processing applied to it. The methods of processing have changed and hence the image and therefore the thresholds have changed? Unless I'm missing something.

Reply:
Makes sense to me that the initial image has changed (and this image from the previous `minicircle_initial_median_flatten` (the first in the sequence) which also zero-means the data hence the threshold is dramatically reduced.